chronograf/1.10/entrypoint.sh

```diff
@@ -9,4 +9,8 @@ if [ "$1" = 'chronograf' ]; then
   export BOLT_PATH=${BOLT_PATH:-/var/lib/chronograf/chronograf-v1.db}
 fi
 
-exec "$@"
+if [ "$(id -u)" -ne 0 ] || { [ ! -z "${CHRONOGRAF_AS_ROOT}" ] && [ "${CHRONOGRAF_AS_ROOT}" != "false" ]; }; then
+    exec "$@"
+else
+    exec setpriv --reuid chronograf --regid chronograf --init-groups "$@"
```

Comment:
I don't see where you updated the non-alpine Dockerfile to add the `chronograf` user and group.

Reply:
The `chronograf` user is created during the package installation process, from the script: https://github.com/influxdata/chronograf/blob/master/etc/scripts/pre-install.sh.

Reply:
Ah, we're installing the deb in the non-alpine version and a tarball in the alpine version. Thanks.